<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DESdialog</class>
 <widget class="QDialog" name="DESdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>D:/Test/DES Encryption.wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D:/Test/DES Decryption.wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D:/Test/DESCipherTest.wav</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browsePButton_2">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>D:/Test/DES Encryption.wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D:/Test/DES Decryption.wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D:/Test/DESCipherTest.wav</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browsePButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Encryption format</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="ECB_RB">
          <property name="text">
           <string>ECB (Electronic Codebook)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="CBC_RB">
          <property name="text">
           <string>CBC (Cipher Block Chaining)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="CFB_RB">
          <property name="text">
           <string>CFB (Cipher Feedback)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Key (HEX)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>0x0123456789ABCDEF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Initialization vector (HEX)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="initVectorEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0xCE27CB141098FEB0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Execution time: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="encryptPB">
           <property name="text">
            <string>Encrypt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="decryptPB">
           <property name="text">
            <string>Decrypt</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Generate Cross-correlation  Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Generate Autocorrelation Code</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder></zorder>
  <zorder>pushButton_2</zorder>
  <zorder>verticalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ECB_RB</sender>
   <signal>toggled(bool)</signal>
   <receiver>initVectorEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DESdialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
